lib/shift/circuit_breaker/circuit_handler.rb

```diff
@@ -13,34 +13,39 @@ module CircuitBreaker
     # the defined skip_duration, ie. no operations are executed and the provided fallback is called.
     #
     class CircuitHandler
-      attr_accessor :name, :error_threshold, :skip_duration, :exception_classes, :error_count, :last_error_time, :state, :logger, :monitor
+      attr_accessor :name, :error_threshold, :skip_duration, :exception_classes, :enable_error_logging, :error_count
+      attr_accessor :last_error_time, :state, :logger, :monitor
 
       DEFAULT_EXCEPTION_CLASSES = [Net::OpenTimeout, Net::ReadTimeout, Faraday::TimeoutError, Timeout::Error].freeze
+      DEFAULT_ERROR_LOGGING_STATE = true.freeze
```

Comment:
Do you need to freeze a boolean when it's set against a constant? Asking because I don't know.